src/dodal/beamlines/i19_1.py

```diff
@@ -35,6 +35,16 @@
 DISPLAY_CONFIG = "/dls_sw/i19-1/software/daq_configuration/domain/display.configuration"
 
 
+# Needs to wait until enum is fixed on the beamline
+# See https://github.com/DiamondLightSource/dodal/issues/1150
+# @device_factory()
```

Comment:
Could: You can just do:
```suggestion
@device_factory(skip=True)
```
Then un-comment the function